project/src/main/webapp/candidate-page/candidate-page.template.html

```diff
@@ -1,18 +1,20 @@
 
 <div class="candidate-page-info" id="official-info-container">
   <h3>{{officialInfo.candidateName}}</h3>
-  <img src={{officialInfo.photoURL}} 
+  <img ng-show="officialInfo.photoURL" src={{officialInfo.photoURL}} 
       alt="Picture of {{officialInfo.candidateName}}">
-  <ul>
+  <ul class="list-unstyled">
     <li>Position: {{officialInfo.position}}</li>
     <li>Party Affiliation: {{officialInfo.partyAffiliation}}</li>
-    <li>Email: {{officialInfo.email}}</li>
-    <li>Phone number: {{officialInfo.phoneNumber}}</li>
+    <li ng-show="officialInfo.email">Email: {{officialInfo.email}}</li>
+    <li ng-show="officialInfo.phoneNumber">
+        Phone number: {{officialInfo.phoneNumber}}</li>
   </ul>
   <p ng-if="officialInfo.isIncumbent"> {{officialInfo.candidateName}}
         currently holds office as {{officialInfo.position}}</p>
-  <p>Click <a href={{officialInfo.website}}>here </a>to be redirected
-      to {{officialInfo.candidateName}}'s website.</p>
+  <p ng-show="officialInfo.website">Click <a href={{officialInfo.website}}>
+      here </a>to be redirected to {{officialInfo.candidateName}}'s website.
```

Comment:
Can you change this to "Candidate's website" and add the hyperlink to the website?

Reply:
Definitely. I can do that pretty easily.